grakn-test/src/test/java/ai/grakn/test/engine/GraknEngineServerIT.java

```diff
@@ -98,8 +97,7 @@ public void whenSendingTasksToTwoEngines_TheyAllComplete(
         assertEquals(completableTasks(allTasks), completedTasks());
     }
 
-    @Ignore  // TODO: Fix this test - may be a race condition
-    @Property(trials=10)
```

Comment:
I can't remember if I deliberate removed the `(trials=10)` - can you remember @alexandraorth ?